api/src/models/project-view.ts

```diff
@@ -26,6 +28,9 @@ export class GetProjectData {
     this.start_date = projectData?.start_date || '';
     this.end_date = projectData?.end_date || '';
     this.comments = projectData?.comments || '';
+    this.completion_status =
+      (projectData && projectData.end_date && moment(projectData.end_date).isBefore(new Date()) && 'Completed') ||
```

Comment:
Not a necessary change by any means:

Instead of `new Date()` you can also just say `moment()`

Is there a difference in the end? I assume not.

Reply:
Oh, do we need to account for when the end_date is today? Maybe a question for the business: do they consider end_date = July 1st to mean 11:59pm on July 1st, or 12:01am on July 1st.  I'd assume the former?

In any case, moment has a `isSameOrBefore` function, if needed.

Reply:
that makes sense :) thanks